hooks/pre/20-certs_update.rb

```diff
@@ -0,0 +1,66 @@
+require 'fileutils'
+require 'English'
+
+if module_enabled?('katello')
+  SSL_BUILD_DIR = param('certs', 'ssl_build_dir').value
+  CHECK_SCRIPT = `which katello-certs-check`.strip
```

Comment:
Why do we need `which`? Isn't it sufficient to use a non-qualified path later on?

Reply:
I tested using the non-qualified path and it and it seems to work just fine